src/component/ui/cards/BookmarksCards.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import './bookmarksCards.scss';
+import Glow from '../buttons/glow/Glow.jsx';
+
+
+const BookmarksCards = props => (
+  <Fragment>
+    <div className='col-md-12'>
+      <div className={`bookmarks-cards ${props.moreClass}`}>
+        <div className='col-md-12'>
+          <div className='header'>
+            <h1 className="col-md-11 card-title">{ props.articleTitle }</h1>
+            <img className='bookmark-logo col-md-1' src='/src/images/bookmark.svg' alt='bookmark-logo' />
+          </div>
+        </div>
+        <div className='col-md-12'>
+          <p className='text-content'>{ props.article}</p>
+        </div>
+        <br />
+        <div className="user-section ml-10 card-bottom">
+          <div className='col-md-12'>
+            <span><img className="user-image" src={props.userImage} alt="user" /></span>
+            <p className="col-8 col-md-9 username" style={{ display: 'inline-block' }}>{ props.username }</p>
+            <span><Glow className="col-1 glowButton" active handleGlow={() => {}} /></span>
+            <span className='glow-text'>{props.glowCount}</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </Fragment>
+);
+
+
+const article = `Lorem ipsum dolor sit amet consectetur adipisicing elit.
```

Comment:
Hello Victor, I think these defaults should be passed in as props. I don't think we should be having default values for article text, username, and title. The image can be an avatar just in case the user does not have a profile image.